quinn-proto/src/connection/mod.rs

```diff
@@ -1011,6 +1011,11 @@ where
         self.path.remote
     }
 
+    /// Current best estimate of this connection's latency (round-trip-time)
+    pub fn rtt(&self) -> Duration {
+        self.path.rtt.conservative()
```

Comment:
Why pick `conservative()` here, rather than `get()`? As I understand it, the latter represents our best estimate. (If we decide to stick with `conservative()`, we should probably elaborate on this in the docstring.

Reply:
I wasn't sure about which to use. I didn't think about it much, I just saw that `conservative` was marked `pub` and `get` wasn't, so I assumed that `get` isn't intended to be used directly.

But I think you are right. `get` is the more correct thing to use. I'll update this.